hpilo_fw.py

```diff
@@ -62,10 +62,10 @@ def download(ilo, path=None, progress = lambda txt: None):
     return False
 
 def parse(fwfile, ilo):
-    fd = open(fwfile)
+    fd = open(fwfile, 'rb')
     data = fd.read()
     fd.close()
-    if '_SKIP=' in data:
+    if b'_SKIP=' in data:
```

Comment:
I'm not familiar with python 2.4--what's the correct way to specify forwards-compatible byte strings? We could do something like:

```python
if '_SKIP='.encode('utf-8') in data:
```

But that seems rather hacky.